### PR TITLE
New features and enhancements 

### PR DESCRIPTION
**Pull request by mihaildanilov**
## New features:
- **Newsletter Form:** Added a subscription form to increase user engagement. This change addresses and resolves issue #60.
- **Next.js Upgrade:** Updated project framework to version 14.2.3, leveraging the latest features for improved performance and resolving issue #66.
- **File Renaming:** Standardized file names across the project to enhance clarity and consistency, improving maintainability.

### CODE REVIEW

File: frontend/messages/lv.json [hunk 1]
```diff
@@ -193,5 +193,14 @@
       "support": "/images/stories/lv/support-lv.webp",
       "friends": "/images/lv/draugi.webp"
     }
+  },
+  "Newsletter": {
```
Comment (oakw) [inline frontend/messages/lv.json:197]:
```  "Newsletter": {
    "header": "Piesakies jaunumiem!",
    "description": "Abonē, lai uzzinātu par gaidāmajiem hakatoniem, darbnīcām un daudz ko citu!",
    "placeholderTitle": "E-pasts",
    "additionalInfo": "*aizpildot šo veidlapu, tu piekrīti saņemt mūsu e-pasta ziņojumus. Jebkurā laikā vari anulēt savu abonementu.",
    "errorMessage": "Lūdzu, ievadi derīgu e-pasta adresi.",
    "buttonText": "Abonēt",
    "listUUID": "f0ac2134-a0e1-4de0-b43f-f56f531102d0"
    

When creating new texts further, stick to the singular form ('tu'/'tev'/'tavs')

**Review by oakw — CHANGES_REQUESTED** [review]
Great, just some minor things. You may also fix that `.env` comment in a separate PR.

**Comment by mihaildanilov** [comment]
Done, all changes have been applied as requested.

**Review by oakw — CHANGES_REQUESTED** [review]
Screenshot_20240513-212948

The padding seems to be missing, especially on smaller screens

**Review by oakw — APPROVED** [review]

**Pull request merged**